.github/workflows/sycl-nightly.yml

```
@@ -217,6 +217,9 @@ jobs:
   run-sycl-cts-linux:
     needs: [ubuntu2204_build, build-sycl-cts-linux]
     if: ${{ always() && !cancelled() && needs.ubuntu2204_build.outputs.build_conclusion == 'success' }}
+    # Normally these jobs take less than 10m. But sometimes the job hangs up and
+    # reaches the 360m limit. Set a lower limit to free up the runner earlier.
+    timeout-minutes: 60
```

```suggestion
    timeout-minutes: 30
```

if it's expected to finish under 10.

---

I wish we could do that, but sometimes (like 3 days ago) it took 32min - https://github.com/intel/llvm/actions/runs/15081117059
So 60m probably would be a better limit